ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>minor nit: add periods to the end of the first two bullets</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>I removed the final period instead and rewrote the first bullet. These bullets aren't full sentences and don't require periods. I rewrote the first bullet to observe parallel structure.</p>